ProjectEureka/project-eureka-flutter
# PE-37 | Fixed Side Menu and added styling to the UI

What I Did:

- Fixed the side menu. It is now accessible for all of the screens: `Home`, `Profile`, `Create New Post`, and `Settings`.
- I also added some styling to the side menu as well as change the color of the **appBar** of each screens. 

How to Test: 

- Click on the `Side Menu` from the `Home` screen and see if it expands. 
- Click on each screen and see if it redirects you to that specific screen you selected.
- On each screen, the `Side Menu` should be accessible.
